decidim-admin/app/commands/decidim/admin/bulk_action.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Admin
+    class BulkAction < Decidim::Command
+      # Public: Initializes the command.
+
+      def initialize(user, action, reportables)
```

Comment:
```suggestion
      def initialize(user, action, moderations)
```
Here we will pass moderations, not reportables. This will give more flexibility.

Reply:
Done